internal/pkg/flink/types/statements.go

```diff
@@ -73,10 +75,26 @@ type MockStatementResult struct {
 type StatementError struct {
 	Msg              string
 	HttpResponseCode int
+	FailureMessage   string
+	Usage            []string
 }
 
 func (e *StatementError) Error() string {
-	return e.Msg
+	if e == nil {
+		return ""
+	}
+	errStr := "Unknown error."
+	if e.Msg != "" {
+		errStr = fmt.Sprintf("Error: %s.", e.Msg)
+	}
+	if len(e.Usage) > 0 {
+		errStr += fmt.Sprintf("\nUsage: %s.", strings.Join(e.Usage, " or "))
```

Comment:
We have `utils.ArrayToCommaDelimitedString()` for this!

Reply:
Using that now!